gui/gui/inc/TGDoubleSlider.h

```diff
@@ -114,29 +114,54 @@ class TGDoubleSlider : public TGFrame, public TGWidget {
 
    virtual void  SetScale(Int_t scale) { fScale = scale; }
    virtual void  SetRange(Float_t min, Float_t max) {
+      SetRangeD((Double_t) min, (Double_t) max);
+   }
+
+   virtual void SetPosition(Float_t min, Float_t max) {
+      SetPositionD((Double_t) min, (Double_t) max);
+   }
+
+   virtual Float_t GetMinPosition() const {
+      return (Float_t) GetMinPositionD();
+   }
+   virtual Float_t GetMaxPosition() const {
+      return (Float_t) GetMaxPositionD();
+   }
+   virtual void GetPosition(Float_t &min, Float_t &max) const {
+      if (fReversedScale) { min = (Float_t)(fVmin+fVmax-fSmax); max = (Float_t)(fVmin+fVmax-fSmin); }
+      else { min = (Float_t)fSmin; max = (Float_t)fSmax; }
+   }
+   virtual void GetPosition(Float_t *min, Float_t *max) const {
+      if (fReversedScale) { *min = (Float_t)(fVmin+fVmax-fSmax); *max = (Float_t)(fVmin+fVmax-fSmin); }
+      else { *min = (Float_t)fSmin; *max = (Float_t)fSmax; }
+   }
+
+   // double precision methods
+
+   virtual void  SetRangeD(Double_t min, Double_t max) {
       fVmin = min; fVmax = max;
       FixBounds(fVmin, fVmax);
    }
 
-   virtual void SetPosition(Float_t min, Float_t max) {
+   virtual void SetPositionD(Double_t min, Double_t max) {
```

Comment:
What lead you to select a new name rather than overloading?

Reply:
Good point, I don't know...

Reply:
Oh yes, it was to prevent this kind of error in existing (users') code:
```
TFitEditor.cxx(2915,12): error C2668: 'TGDoubleSlider::SetPosition': ambiguous call to overloaded function
```

Reply:
I see ... this is because the input in an integer ... so another option is to add a 3rd overload .. or just leave it as you have it.

Reply:
Thanks @pcanal I'll make some tests and see what is the best solution

Reply:
@pcanal Well, with more overloads come different potential errors:
```
TParallelCoordEditor.cxx(988,66): error C2666: 'TGDoubleSlider::SetRange': 4 overloads have similar conversions
TTreeViewer.cxx(1148,48): error C2666: 'TGDoubleSlider::SetRange': 4 overloads have similar conversions
TTreeViewer.cxx(1149,51): error C2666: 'TGDoubleSlider::SetPosition': 4 overloads have similar conversions
TTreeViewer.cxx(2839,45): error C2666: 'TGDoubleSlider::SetRange': 4 overloads have similar conversions
TTreeViewer.cxx(2840,48): error C2666: 'TGDoubleSlider::SetPosition': 4 overloads have similar conversions
```

Reply:
For the record, those are failing because of the argument are of ```(int, long long)``` ...
and still for the record even std::max is challenged by that setup:
```
#include <algorithm>
  
void func() {
 int i;
 long long l;
 std::max(i, l);
}
```
```
/var/tmp/tmp.cxx:6:2: error: no matching function for call to 'max'
 std::max(i, l);
 ^~~~~~~~
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/algorithm:2633:1: note: candidate template ignored: deduced conflicting types for parameter '_Tp' ('int' vs. 'long long')
max(const _Tp& __a, const _Tp& __b)
^

```

Reply:
So a solution to make the "more overload" work is simply to case the `0` n those 5 places into a long long.

Reply:
@pcanal OK, I added a couple of overloads, can you approve this PR?